dbt_coverage/__init__.py

```diff
@@ -166,57 +138,45 @@ def from_nodes(cls, manifest_nodes: Dict[str, Dict]) -> Manifest:
         """Constructs a ``Manifest`` by parsing from manifest.json nodes."""
 
         sources = [
-            table
-            for table in manifest_nodes.values()
-            if table["resource_type"] == "source"
+            table for table in manifest_nodes.values() if table["resource_type"] == "source"
         ]
         sources = {
-            cls._full_table_name(table): {
+            table["unique_id"]: {
                 "columns": cls._normalize_column_names(table["columns"]),
                 "original_file_path": cls._normalize_path(table["original_file_path"]),
-                "unique_id": table["unique_id"],
+                "name": cls._full_table_name(table),
```

Comment:
We made the method return `_full_table_name` again and used this as `name` here.